src/ApiService/ApiService/model.cs

```diff
@@ -1,87 +1,87 @@
-using Azure.Data.Tables;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
 using System;
-using System.Runtime.Serialization;
+using System.Collections.Generic;
 
-namespace ApiService;
+namespace Microsoft.OneFuzz.Service;
 
 
-record NodeCommandStopIfFree { }
+/// Convention for database entities:
+/// All entities are represented by immutable records
+/// All database entities need to derive from EntityBase
+/// Only properties that also apears as parameter initializers are mapped to the database
+/// The name of the property will be tranlated to snake case and used as the column name
+/// It is possible to rename the column name by using the [property:JsonPropertyName("column_name")] attribute
+/// the "partion key" and "row key" are identified by the [PartitionKey] and [RowKey] attributes
+/// Guids are mapped to string in the db
```

Comment:
is this because of python code behaving same way ? Guids are strings ?

Reply:
yes that's how the python code stores guids